src/fglm/linalg-fglm.c

```diff
@@ -623,7 +624,169 @@ static inline void _4mul_new_matrix_vector_product(uint32_t* vec_res, const uint
 AVX2-based matrix vector product
 **/
 
+#define PARALLEL 1
```

Comment:
This seems a bit dangerous at first, but maybe not when we look at what is inside the function (there does not seem to be anything which will break when non-parallel is wanted). Why keeping two versions of the function?

Reply:
I don't understand this: We have the macro _OPENMP which defines when parallelization is available in msolve. There is no need to introduce another macro. Moreover, there is no need at all to keep two versions of this function.

Reply:
Thanks! This has been cleaned up and fixed.